docs/excel/excel-add-ins-workbooks.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Work with workbooks using the Excel JavaScript API
 description: ''
-ms.date: 02/28/2019
+ms.date: 04/22/2019
```

Comment:
Due to my tardiness on reviewing this, we probably should update the ms.date value.